code/modules/clothing/masks/miscellaneous.dm

```diff
@@ -18,7 +20,42 @@
 /obj/item/clothing/mask/muzzle/attack_hand(mob/user as mob)
 	if(user.wear_mask == src && !user.IsAdvancedToolUser())
 		return 0
+	else if(user.wear_mask == src && security_lock && locked)
+		if(ishuman(user))
+			if(doUnlock(user))

```

Comment:
Oh right, glossed over the security lock part in the preceding conditional. Probably should have paid more attention but I was on my way out the door at that point.